opentok/opentok-meet
# Wait until we're connected before checking subscriber stats

#### What is this PR doing?

Wait until we get the subscriber connected event before we start calling getStats so we don't get those annoying errors about calling getStats when the subscriber is not connected.

#### How should this be manually tested?

Open up 2 tabs so that they're subscribing to one another. Open up the console and make sure there is no error "Cannot call getStats before there is a connection."

#### What are the relevant tickets?

[OPENTOK-33694](https://tokbox.atlassian.net/browse/OPENTOK-33694)
